main.tf

```
@@ -203,6 +203,10 @@ data "aws_region" "this" {}
 data "aws_caller_identity" "this" {}
 
 locals {
+  // Config aggregation isn't enabled for ap-northeast-3
+  // https://docs.aws.amazon.com/config/latest/developerguide/aggregate-data.html
+  is_enabled = module.this.enabled && data.aws_region.this.name != "ap-northeast-3"
```

We usually use `local.enabled`

```suggestion
  enabled = module.this.enabled && data.aws_region.this.name != "ap-northeast-3"
```

---

0620d53caac0c45d21d53824c41ffaadbacf5ad2